### PR TITLE
fix some readme

### PR DESCRIPTION
**Pull request by CaitinZhao**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Sort out the logic of the readme.

### CODE REVIEW

File: .idea/workspace.xml
Comment (hadipash) [inline .idea/workspace.xml:1]: remove this file

File: README.md [hunk 1]
```diff
@@ -107,19 +111,21 @@ python tools/train.py --config {path/to/model_config.yaml}
 
 The `--config` arg specifies the path to a yaml file that defines the model to be trained and the training strategy including data process pipeline, optimizer, lr scheduler, etc.
 
-MindOCR provides SoTA OCR models with their training strategies in `configs` folder.
+MindOCR provides SOTA OCR models with their training strategies in `configs` folder.
```
Comment (SamitHuang) [inline README.md:114]: same

File: README.md [hunk 2]
```diff
@@ -107,19 +111,21 @@ python tools/train.py --config {path/to/model_config.yaml}
...
 python tools/train.py --config configs/det/dbnet/db++_r50_icdar15.yaml
 ```
+
 ```shell
 # train text recognition model CRNN on icdar15 dataset
 python tools/train.py --config configs/rec/crnn/crnn_icdar15.yaml
 ```
 
-Similarly, it is simple to evaluate the trained model with the `tools/eval.py` script.
+Similarly, it is a simple to evaluate the trained model with the `tools/eval.py` script.
```
Comment (SamitHuang) [inline README.md:127]: `it is a simple` -> `it is easy`

File: docs/en/inference/convert_tutorial.md [hunk 3]
```diff
@@ -1,17 +1,32 @@
 ## Inference - Model Conversion Tutorial
 
+MindOCR Inference support infer from MindOCR trained models and third-party(PaddelOCR and mmOCR) models.
```
Comment (SamitHuang) [inline docs/en/inference/convert_tutorial.md:3]: MindOCR supports model inference for models trained using MindOCR and models converted from third-party toolboxes (PaddelOCR and MMOCR).

File: docs/en/inference/convert_tutorial.md [hunk 4]
```diff
@@ -166,13 +181,35 @@ paddle2onnx \
 The `input_shape_dict` in the parameter can generally be viewed by opening the inference model using the [Netron](https://github.com/lutzroeder/netron),
 or found in the code in [tools/export_model. py](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/tools/export_model.py) above.
 
-#### 2.3 ONNX -> OM
+#### 2.3 ONNX -> MindIR
+
+The `converter_lite` can be used to convert the ONNX into a MindSpore Lite MindIR. For detailed usage tutorials,
```
Comment (SamitHuang) [inline docs/en/inference/convert_tutorial.md:186]: The `converter_lite` script can be used to convert an ONNX model to a MindIR model.

File: docs/en/inference/convert_tutorial.md [hunk 5]
```diff
@@ -166,13 +181,35 @@ paddle2onnx \
...
+    --NoFusion=false \
+    --fmk=ONNX \
+    --device=Ascend \
+    --modelFile=det_db.onnx \
+    --outputFile=det_db_output \
+    --configFile=config.txt
+```
+
+The conversion process is completely the same as the [MindOCR models](#1-mindocr-models),
+except that `--fmk` needs to specify that the input is the ONNX, which will not be repeated here.
```
Comment (SamitHuang) [inline docs/en/inference/convert_tutorial.md:203]: `--fmk` --> `--fmt` ??

except that `--fmt`  needs to be set as `ONNX`.

File: docs/en/inference/convert_tutorial.md [hunk 6]
```diff
@@ -1,17 +1,32 @@
...
 ```mermaid
 graph LR;
-    ckpt --> |export| MindIR --> |"converter_lite(offline conversion)"| o[MindIR];
+    A[MindOCR models] -- export --> B[MindIR] -- converter_lite --> C[MindSpore Lite MindIR];
 ```
 
 #### 1.1 Model Export
 
-Before inference, it is necessary to [export](https://github.com/mindspore-lab/mindocr/blob/main/configs/README.md) the trained ckpt to a MindIR file, which stores the model structure and weight parameters.
+Before inference, it is necessary to export the trained ckpt to a [MindIR](https://www.mindspore.cn/docs/zh-CN/r2.0.0-alpha/design/mindir.html) file. Please run `tools/export.py`.
```
Comment (hadipash) [inline docs/en/inference/convert_tutorial.md:16]: change to the English link:
https://www.mindspore.cn/docs/en/r2.0/design/mindir.html

File: docs/en/inference/convert_tutorial.md [hunk 7]
```diff
@@ -107,15 +122,15 @@ If not set, defaults to `enforce_fp16`.
 
 ### 2. PaddleOCR models
 
-The PaddleOCR models support two inference backends: [ACL](https://www.hiascend.com/document/detail/zh/canncommercial/63RC1/inferapplicationdev/aclcppdevg/aclcppdevg_000004.html) and [MindSpore Lite](https://www.mindspore.cn/lite),
-corresponding to the OM model and MindIR model, respectively.
+The PaddleOCR models support two inference backends: [MindSpore Lite](https://www.mindspore.cn/lite) and [ACL](https://www.hiascend.com/document/detail/zh/canncommercial/63RC1/inferapplicationdev/aclcppdevg/aclcppdevg_000004.html),
```
Comment (hadipash) [inline docs/en/inference/convert_tutorial.md:125]: here as well: https://www.mindspore.cn/lite/en
https://www.hiascend.com/document/detail/en/CANNCommunityEdition/600alphaX/infacldevg/aclcppdevg/aclcppdevg_000003.html

File: docs/en/inference/convert_tutorial.md [hunk 8]
```diff
@@ -166,13 +181,35 @@ paddle2onnx \
...
+    --saveType=MINDIR \
+    --NoFusion=false \
+    --fmk=ONNX \
+    --device=Ascend \
+    --modelFile=det_db.onnx \
+    --outputFile=det_db_output \
+    --configFile=config.txt
+```
+
+The conversion process is completely the same as the [MindOCR models](#1-mindocr-models),
```
Comment (hadipash) [inline docs/en/inference/convert_tutorial.md:202]: The conversion process is the same as for ...

**Review by hadipash — APPROVED** [review]

File: docs/en/inference/convert_tutorial.md (hunk 5 above)
Comment (CaitinZhao) [reply docs/en/inference/convert_tutorial.md:203]: the original format of the input model is fmk in lite converter_tool. https://www.mindspore.cn/lite/docs/en/r2.0/use/converter_tool.html

**Review by SamitHuang — APPROVED** [review]

**Comment by SamitHuang** [comment]
lgtm

**Pull request merged**